wiki/Beatmap/Hype/de.md

```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+tags:
+  - train
+---
+
+# Hype
+
+Jede [ausstehende](/wiki/Beatmap/Category#work-in-progress-und-ausstehend) [Beatmap](/wiki/Beatmap) hat einen **Hype-Train**, der grob zeigt, wie viele Spieler am [Ranking](/wiki/Beatmap/Category#ranked) der Beatmap interessiert sind. Nutzer können spezielle [Beatmapdiskussionen](/wiki/Beatmap_discussion) im Tab `Generell (alle Schwierigkeitsstufen)` erstellen und die Option `Hype!` verwenden, um zum Hype-Train beizutragen.
```

Comment:
```suggestion
Jede [ausstehende](/wiki/Beatmap/Category#work-in-progress-und-ausstehend) [Beatmap](/wiki/Beatmap) hat einen **Hype-Train**, der grob zeigt, wie viele Spieler daran interessiert sind, die Map als [Ranked](/wiki/Beatmap/Category#ranked) zu sehen. Nutzer können spezielle [Beatmap-Diskussionen](/wiki/Beatmap_discussion) im Tab `Generell (alle Schwierigkeitsstufen)` erstellen und die Option `Hype!` verwenden, um zum Hype-Train beizutragen.
```

Reply:
Sticking this close to the English wording hurts it a bit IMO. 
How about:

```suggestion
Jede [ausstehende](/wiki/Beatmap/Category#work-in-progress-und-ausstehend) [Beatmap](/wiki/Beatmap) hat einen **Hype-Train**, der grob zeigt, wie viele Spieler daran interessiert sind, dass die Beatmap [gerankt](/wiki/Beatmap/Category#ranked) wird. Nutzer können spezielle [Beatmap-Diskussionen](/wiki/Beatmap_discussion) im Tab `Generell (alle Schwierigkeitsstufen)` erstellen und die Option `Hype!` verwenden, um zum Hype-Train beizutragen.
```

Reply:
sounds good to me